test/README.md

```diff
@@ -0,0 +1,7 @@
+# Libnxz tests
+
+This directory contains test cases.
+To be compliant with zlib, usually we use libnxz to deflate,
+the compressed data will use both zlib and libnxz to inflate for verification.
+We also use zlib to inflate, the compressed data will use libnxz to deflate
```

Comment:
This is the same information as the lines above, here we should say zlib to **deflate** and libnxz to **inflate**.

Reply:
@rzinsly Fixed.